### ISSUE TITLE
on sale in my store

### ISSUE THREAD

**Issue by danielb2** [twaddington/django-olcc#10]
Is it possible to get all the liquors on sale which are available at my specific store?

**Comment by twaddington** [comment]
Could you elaborate? My understanding is that OLCC sale prices are state wide. Are you looking for inventory at your local store?

**Comment by danielb2** [comment]
Yes, inventory at local store which has items on sale. What happens is I see the on-sale prices, I have to copy and paste the product id into oregonliquorsearch.com to see if the inventory is there.

For example:

http://www.oregonliquorprices.com/products/0649c/

means I have to search for `0649C` and specify my local store etc.

It's a rather sucky process.

It would be useful if I could select a default location too and have it show things available in the vincinity.

http://oregonliquorsearch.com/servlet/FrontController?view=global&action=search&radiusSearchParam=10&productSearchParam=0649C&locationSearchParam=97701&btnSearch=Search

This is my result for this particular product.

In addition, it would be excellent ot be able to sort the results on pricing.

**Comment by twaddington** [comment]
That's a great suggestion.

If the OLCC published data on current store inventories I could easily add this to my site. However, it doesn't appear they provide this info in a way I could easily import it.

For example, I import the monthly price data from this CSV (comma separated values) file every month:

http://www.olcc.state.or.us/pdfs/NumericPriceListNextMonth.csv

Unfortunately there's no document that I can find containing inventory data.

I'll send them an email and see if I can work something out.

**Comment by danielb2** [comment]
Wow, love how they have a csv in a directory labeled `pdfs`

**Comment by danielb2** [comment]
btw, something you can do in the meantime, is like the item ID to a selected store, and redirect it to oregonliquorsearch so it's at least just a one click ordeal

**Comment by twaddington** [comment]
Yeah, I was thinking that could be a good option. I'll look into adding a link. In the meantime I also sent an email to the OLCC requesting that data. I've had success with this in the past as it's how I originally imported the store info.
